nbi/engine/src/org/netbeans/installer/utils/system/launchers/impl/CommonLauncher.java

```diff
@@ -95,8 +95,6 @@ protected long addData(FileOutputStream fos, File file, Progress progress, long
     
     //add rnd data
     protected void addData(FileOutputStream fos) throws IOException {
-        double rand = Math.random() * Byte.MAX_VALUE;
-        //fos.write(new byte[] {(byte)rand});
         fos.write(new byte[] {'#'});
     }
```

Comment:
i don't know why this is commented out but the method does not make sense without it

Reply:
Because rand  unused and redundant multiplication operation

Reply:
I see that it is unused. I am wondering why. because the comment above the says "add rnd data". So we have to find out what the method is actually supposed to do before removing anything.

Reply:
best approach would be to simply leave it as is or comment it out. Maybe someone wants to use the other version in future.